### DISCUSSION TITLE
Is there a natural convection inlet boundary?

### DISCUSSION THREAD

**Discussion by tuanhtrng** [Nek5000/nekRS#422] [Q&A]
What is the best way to implement an inlet that counters the outflow due to natural convection?

I tried velRecycling (using turbPipe example) but the pressure solver explodes. Damping the negative velocity parallel to the flow direction would keeps the code running but it actually turns the problem into a mixed convection and eventually becomes a forced convection.

**Comment by stgeke** [comment]
I suggest to ask this non-technical question on the nek5000 mailing list.
